soap/services.py

```
@@ -0,0 +1,87 @@
+        f"## Objective (معاینه)\n\n"
+        f"**علائم حیاتی:**\n"
+        f"- فشار خون: {soap_report['objective'].get('vital_signs', {}).get('bp', 'ثبت نشده')}\n"
+        f"- ضربان قلب: {soap_report['objective'].get('vital_signs', {}).get('hr', 'ثبت نشده')}\n"
+        f"- تنفس: {soap_report['objective'].get('vital_signs', {}).get('rr', 'ثبت نشده')}\n"
+        f"- دما: {soap_report['objective'].get('vital_signs', {}).get('temp', 'ثبت نشده')}\n\n"
+        f"**معاینه فیزیکی:**\n"
+        f"{soap_report['objective'].get('physical_exam', 'انجام نشده')}\n\n"
+        f"## Assessment (ارزیابی)\n\n"
+        f"## Plan (برنامه درمان)\n\n"
+    )
+    return md
```

Ruff (0.12.2)

Unnecessary assignment to `md` before `return` statement

Remove unnecessary assignment

(RET504)